app/controllers/coins_controller.rb

```diff
@@ -59,8 +91,16 @@ def get_coin
   end
 
   def calc_hours_date_to_now(date)
-    (Time.now - date)/1.hours
+    (Time.now - date) / 1.hours
+  end
+
+  def name_network
+    @network = Network.all.pluck(:name)
```

Comment:
```suggestion
    @network_names = Network.all.pluck(:name)
```
đặt tên biến dạng gợi nhớ